internal/storage/db/internal/db.go

```diff
@@ -655,29 +655,41 @@ func (s *dbStorage) Delete(ctx context.Context, ids ...namer.ModuleID) error {
 	return nil
 }
 
-func (s *dbStorage) InspectPolicies(ctx context.Context, listParams storage.ListPolicyIDsParams) (map[string]*responsev1.InspectPoliciesResponse_Result, error) {
-	policyIDs, err := s.ListPolicyIDs(ctx, listParams)
+func (s *dbStorage) InspectPolicies(ctx context.Context, params storage.InspectPoliciesParams) (map[string]*responsev1.InspectPoliciesResponse_Result, error) {
+	policyIDs, err := s.ListPolicyIDs(ctx, storage.ListPolicyIDsParams{
```

Comment:
We shouldn't make filtering by IDs a special case. It can be just another filtering option in the same vein as `NameRegexp`, `IncludeDisabled` etc. 

The list of IDs should be part of the database query. Otherwise, if no other filters are defined, we'll be loading everything in the database to memory and filtering them out -- which is really inefficient.

Reply:
> We shouldn't make filtering by IDs a special case. It can be just another filtering option in the same vein as NameRegexp, IncludeDisabled etc.

Then maybe we could add `IDs` filtering option to the `storage.ListPolicyIDsParams` and use it in the `InspectPolicies` like used to. This way `ListPolicyIDs` could support filtering by IDs too.

Reply:
Yes. Sorry, I should have been clear about it. I thought it was obvious.